types/bull-arena/index.d.ts

```diff
@@ -1,56 +1,61 @@
-// Type definitions for bull-arena 2.6
+// Type definitions for bull-arena 3.0
 // Project: https://github.com/bee-queue/arena/
 // Definitions by: Levi Bostian <https://github.com/levibostian>
+//                 Gaurav Sharma <https://github.com/gtpan77>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.8
+// TypeScript Version: 4.1
```

Comment:
What is the reason for incrementing the typescript version? Can we leave it at 2.8?

Reply:
Yes, we can leave it. But, I have tested it on latest version of typescript

Reply:
the CI server will run all the tests. I would put it back to 2.8 and the CI will tell you if all is good.

Reply:
reverted